src/pretix/control/views/vouchers.py

```diff
@@ -215,7 +215,7 @@ def get_form_kwargs(self):
     @transaction.atomic
     def form_valid(self, form):
         form.instance.event = self.request.event
-        messages.success(self.request, _('The new voucher has been created.'))
+        messages.success(self.request, _('The new voucher has been created.') + ' (%s)' % form.instance.code)
```

Comment:
I'd suggest a more translatable form here, as the style with the parantheses might not be appropriate in every language.

```
 messages.success(self.request, _('The new voucher has been created: {}').format(form.instance.code))
```